extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [UNRELEASED]
 
+- Implement syntax highlighting for the `additional` keyword.
```

Comment:
Please link to this PR like other changelog entries have.

Reply:
I have done that in commit 4.